release-content/migration-guides/wayland.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Enable Wayland by default
+pull_requests: [19232]
+---
+
+Wayland has now been added to the default features of the `bevy` crate.
+
```

Comment:
Maybe write something here to "introduce" the reader to the error message before showing it :)